azext_iot/tests/central/test_iot_central_devices_int.py

```diff
@@ -97,7 +123,7 @@ def test_device_connect(self):
         command = "iot central device compute-device-key --pk {} -d {}".format(
             APP_PRIMARY_KEY, device_id
         )
-        device_primary_key = self.cmd(
+        device_primary_key = self.cmd_withoutToken(
```

Comment:
"compute-device-key" does not use api-version as well. consider to create a sort of "cmd_with_no_params" or add "token" as an ignored parameter to the command